make/acceptance.mk

```diff
@@ -62,6 +62,7 @@ test-acceptance: test-acceptance-setup
 	$(Q)TEST_ACCEPTANCE_START_SBO=$(TEST_ACCEPTANCE_START_SBO) \
 		TEST_ACCEPTANCE_SBO_STARTED=$(TEST_ACCEPTANCE_SBO_STARTED) \
 		TEST_NAMESPACE=$(TEST_NAMESPACE) \
+		HACK_DIR=$(HACK_DIR) \
```

Comment:
This change is not necessary now that you removed the finalizers script. It should not be there